queries.yaml

```diff
@@ -11,21 +11,126 @@ all:
 
 sources:
 
-  # NOTE: these are split this way to represent each query that goes to the database
+  # NOTE: each source represents a query to the database
+  # within each source, a list of templates can be used
+  # the templates will be combined with a sql `union all` in between
+  # therefore, it's necessary that the query results of each template
+  # have the same types
+
+  # NOTE: zoom_stop is exclusive
+
   planet_osm_polygon:
-    template: planet_osm_polygon.jinja2
+    - template: planet_osm_polygon.jinja2
   planet_osm_line:
-    template: planet_osm_line.jinja2
+    - template: planet_osm_line.jinja2
   planet_osm_point:
-    template: planet_osm_point.jinja2
+    - template: planet_osm_point.jinja2
+
   ne:
-    template: ne.jinja2
+    ## places
+    # ne_10m_populated_places
+    - template: ne-places.jinja2
+
+    ## boundaries
+
+    # ne_110m_admin_0_boundary_lines_land
+    - template: ne-boundaries-110m.jinja2
+      zoom_stop: 2
+
+    # ne_50m_admin_0_boundary_lines_land
+    # ne_50m_admin_1_states_provinces_lines
+    - template: ne-boundaries-50m.jinja2
+      zoom_start: 2
+      zoom_stop: 5
+
+    # ne_10m_admin_0_boundary_lines_land
+    # ne_10m_admin_0_boundary_lines_map_units
+    # ne_10m_admin_1_states_provinces_lines
+    - template: ne-boundaries-10m.jinja2
+      zoom_start: 5
+      zoom_stop: 8
+
+    ## earth
+
+    # ne_110m_land
+    - template: ne-earth-110m.jinja2
+      zoom_stop: 2
+
+    # ne_50m_land
+    - template: ne-earth-50m.jinja2
+      zoom_start: 2
+      zoom_stop: 5
+
+    # ne_10m_land
+    - template: ne-earth-10m.jinja2
+      zoom_start: 5
+      zoom_stop: 9
+
+    ## landuse
+
+    # ne_50m_urban_areas
+    - template: ne-landuse-50m.jinja2
+      zoom_start: 4
+      zoom_stop: 6
+
+    # ne_10m_urban_areas
+    - template: ne-landuse-10m.jinja2
+      zoom_start: 6
+      zoom_stop: 10
+
+    ## roads
+
+    # ne_10m_roads
+    - template: ne-roads.jinja2
+      zoom_start: 5
+      zoom_stop: 8
+
+    ## water
+
+    # ne_110m_ocean
+    # ne_110m_lakes
+    # ne_110m_coastline
+    # ne_110m_lakes
+    - template: ne-water-110m.jinja2
+      zoom_stop: 2
+
+    # ne_50m_ocean
+    # ne_50m_lakes
+    # ne_50m_coastline
+    # ne_50m_lakes
+    - template: ne-water-50m.jinja2
+      zoom_start: 2
+      zoom_stop: 5
+
+    # ne_10m_ocean
+    # ne_10m_lakes
+    # ne_10m_coastline
+    # ne_10m_lakes
+    - template: ne-water-10m.jinja2
+      zoom_start: 5
+      zoom_stop: 9
+
+    # ne_50m_playas
+    - template: ne-water-playas-50m.jinja2
+      zoom_start: 4
+      zoom_stop: 5
+
+    # ne_10m_playas
+    - template: ne-water-playas-10m.jinja2
+      zoom_start: 5
+      zoom_stop: 9
+
   osmdata:
-    template: osmdata.jinja2
-    start_zoom: 8
+
+    - template: osmdata-earth.jinja2
+      zoom_start: 9
```

Comment:
Nice catch! That explains why the tests were failing :) When I rebased off master I didn't catch that this changed upstream.

Fixed in b8ed8e6.